packages/edge-worker/src/EdgeWorker.ts

```diff
@@ -80,11 +80,15 @@ export class EdgeWorker extends EventEmitter {
 	private ndjsonClients: Map<string, NdjsonClient> = new Map(); // listeners for webhook events, one per linear token
 	private persistenceManager: PersistenceManager;
 	private sharedApplicationServer: SharedApplicationServer;
+	private cyrusHome: string;
 
 	constructor(config: EdgeWorkerConfig) {
 		super();
 		this.config = config;
-		this.persistenceManager = new PersistenceManager();
+		this.cyrusHome = config.cyrusHome || join(homedir(), ".cyrus");
```

Comment:
don't have the fallback, instead, make it mandatory not optional to pass to the edge worker